Essentials/src/config.yml

```diff
@@ -480,11 +480,13 @@ mails-per-minute: 1000
 # Set to -1 to disable, and essentials.tempban.unlimited can be used to override.
 max-tempban-time: -1
 
-# Changes /reply functionality. If true, /r goes to the person you messaged last, otherwise the first person that messaged you. 
```

Comment:
I would add a comment stating "Players can use /rtoggle to change this behaviour for themselves."